onnx/test/test_external_data.py

```diff
@@ -342,49 +342,124 @@ def test_save_model_does_convert_and_saves_the_model(self):  # type: () -> None
         self.assertFalse(attribute_tensor.HasField("data_location"))
         self.assertTrue(np.allclose(to_array(attribute_tensor), self.attribute_value))
 
+    def test_save_model_does_not_load_external_tensor(self):  # type: () -> None
+        model_file_path = self.get_temp_model_filename()
+        onnx.save_model(self.model,
+                        model_file_path,
+                        save_as_external_data=True,
+                        location=None,
+                        size_threshold=0,
+                        convert_attribute=False)
+        # Save without load_external_data
+        model = onnx.load_model(model_file_path, load_external_data=False)
+        onnx.save_model(model,
```

Comment:
do you have a test where both saves will save some tensors as external data? this will validate whether the write to external file during second save is done properly (meaning offsets are honored etc)

During the second save you can turn on attribute serialization to external data to make the test easy

Reply:
Just added a test `test_save_model_with_external_data_and_threshould` to test saving a model with typical tensor and external tensor.